_sass/components/_news-section.scss

```diff
@@ -0,0 +1,102 @@
+.c-latest-news {
+  width: 80%;
+  margin: 4rem auto 3.5rem;
+
+  @media (min-width: $bp-md) {
```

Comment:
I love the use of variables in your media queries!

Reply:
thanks 😎